### PR TITLE
feat: add Story Testnet

### PR DESCRIPTION
**Pull request by atuser14**
### Description
There is missing the trendy story testnet ,so i request to add this to hyperlane registry.

### Backward compatibility

no
### Testing
yes

### CODE REVIEW

File: chains/storytestnet/logo.svg
Comment (jmrossy) [inline chains/storytestnet/logo.svg:1]: Can you please remove the white background around the edges?

Comment (atuser14) [reply chains/storytestnet/logo.svg:1]: i removed the white background,thanks for the review.

Comment (jmrossy) [reply chains/storytestnet/logo.svg:1]: Are you sure? There's a white rectangle in the bottom-right

File: chains/storytestnet/addresses.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+proxyAdmin: "0xAf47409d9ea3A1EDC9611C89999823844C2d54Be"
+staticAggregationHookFactory: "0x2D58cFecEA1ea0fc8cBa99B6e81d2917E9961a7b"
+staticAggregationIsmFactory: "0xfeaE28b3CfE111932fE6D476fC9567E26D5655B3"
+staticMerkleRootMultisigIsmFactory: "0x5819b8e0eb555DD1d58Bb7866Af5d485a4958F60"
+staticMerkleRootWeightedMultisigIsmFactory: "0x17169Ec27a0b9eA5E49b0322F07bAd3cca7E95d2"
+staticMessageIdMultisigIsmFactory: "0xA782825BF7E64824cB4D668B2D66D0EB3984DEdF"
+staticMessageIdWeightedMultisigIsmFactory: "0x051B57EF8eda07A38D3521C33D2cb3D35Edf548E"
+testRecipient: "0x3544cEe39DFdA00CDd196853b3e62eD010758f93"
+validatorAnnounce: "0xa0d54e08aE5B9386C2EdA1319E13f68aA1D4885B"
+TransparentUpgradeableProxy: "0x4E14A4a64A9eEC00EB773891562BABFE3B824530"
```
Comment (xeno097) [inline chains/storytestnet/addresses.yaml:14]: Hey @atuser14 thank you for your PR to the Hyperlane Registry! Could you please remove the `TransparentUpgradeableProxy` field from this file? After that run `yarn lint --fix` to properly format the file and have the fields in the expected order. 

Finally sync your branch with the main branch to be sure that the ci runs on the latest version of the changes

**Review by xeno097 — CHANGES_REQUESTED** [review]

**Comment by jmrossy** [comment]
As JaunePomme said, you need to run `yarn prettier` to fix the formatting issue with your addresses file

**Review by xeno097 — APPROVED** [review]

**Pull request merged**